### DISCUSSION TITLE
Computing exercise 2 empty repo

### DISCUSSION THREAD

**Discussion by PatrickC12** [Part-II-Computational-Physics/.github#23] [General]
Hi all,

I have created the github repo for computing exercise 2, using the link distributed in the email. However, the repo appears to be empty and is asking me to create files within it.

![image](https://user-images.githubusercontent.com/123903514/221154162-2e96bde4-a309-4474-9908-1056bd23d70b.png)

Is there a way to recreate the repo using the link sent by Professor Lamacraft ?

Thanks

**Comment by cl804** [comment]
Same issue here

**Comment by AustenLamacraft** [comment]
Thanks for this. It's fixed now with the link in the second email
